components/trakt/sources/new-show-or-movie-watched/new-show-or-movie-watched.mjs

```diff
@@ -0,0 +1,46 @@
+import trakt from "../../trakt.app.mjs";
+import constants from "../common/constants.mjs";
+import { DEFAULT_POLLING_SOURCE_TIMER_INTERVAL } from "@pipedream/platform";
+
+export default {
+  name: "New Show Or Movie Watched",
+  version: "0.0.1",
+  key: "trakt-new-show-or-movie-watched",
+  description: "Emit new event on each new watched show or movie.",
+  type: "source",
+  dedupe: "unique",
+  props: {
+    trakt,
+    db: "$.service.db",
+    timer: {
+      type: "$.interface.timer",
+      static: {
+        intervalSeconds: DEFAULT_POLLING_SOURCE_TIMER_INTERVAL,
+      },
+    },
+    type: {
+      label: "Type",
+      description: "The content type. E.g. `movies`",
+      type: "string",
+      options: constants.CONTENT_TYPES,
+    },
+  },
+  methods: {
+    emitEvent(data) {
```

Comment:
this method is very alike the one in `new-show-or-movie-rated.mjs`, so you could move it to a common/base file